spec/domain/finders/content_spec.rb

```diff
@@ -235,6 +235,33 @@
   end
 
   describe 'Order' do
+    context 'when there are NULLS' do
```

Comment:
It is not obvious that we are testing the `satisfaction` attribute. I first thought that we were sorting by `title` and having `'null'` in the title was a test issue.

It is a bit tricky because the `satisfaction` will be `null` only when `useful_yes` and `useful_no` are 0. 

```ruby
create :metric, edition: edition1, date: 15.days.ago, useful_yes: 1, useful_no: 0 # satisfaction: 1
create :metric, edition: edition2, date: 15.days.ago, useful_yes: 0, useful_no: 1 # satisfaction: 0
create :metric, edition: edition3, date: 15.days.ago, useful_yes: 0, useful_no: 0 # satisfaction: NULL
```

If Lint is happy with the code above, it could be an option. Although it would be nice to find a different one that does not depend on coments in code.

Reply:
Yup agree - I've added the comments like you suggested. I couldn't see an clearer way. I think because no columns in fact_edition are NULLable, the only way to get a NULL in the aggregation views is indirectly via useful_yes/no.